utils/team_utils.jsx

```diff
@@ -15,14 +11,14 @@ function sortTeamsByDisplayName(a, b) {
 }
 
 // Use to filter out teams that are deleted and without display_name, then sort by their `display_name` field
-export function filterAndSortTeamsByDisplayName(teams = []) {
+export function filterAndSortTeamsByDisplayName(teams, locale) {
     if (!teams) {
         return [];
     }
 
-    return teams.
-        filter((team) => {
-            return team && !team.delete_at > 0 && team.display_name != null;
-        }).
-        sort(sortTeamsByDisplayName);
+    return teams.filter((team) => {
```

Comment:
nit: could just use a expressions here:
```js
return teams.filter((team) => (
    team && team.delete_at > 0 && team.display_name != null
)).sort((a, b) => (
    compareTeamsByDisplayName(locale, a, b)
));

Reply:
0/5 on how we do those. I'm just in the habit of always using a function body when it stretches over multiple lines